source/compiler.ts

```diff
@@ -5,6 +5,7 @@ import stripJsonComments from 'strip-json-comments';
 
 import { i18n } from './common';
 import { isJSONObject, set } from './utils';
+// eslint-disable-next-line no-duplicate-imports
 import type { JSON } from './utils';
```

Comment:
This feels like a ESLint misconfiguration. I think it warrants a deeper look (not necessarily within this PR).